CONTRIBUTING.rst

```diff
@@ -57,121 +52,159 @@ If you are proposing a feature:
 Get Started!
 ------------
 
-Ready to contribute? Here's how to set up `xscen` for local development.
+.. note::
+
+    If you are new to using GitHub and `git`, please read `this guide <https://guides.github.com/activities/hello-world/>`_ first.
+
+.. warning::
+
+    Anaconda Python users: Due to the complexity of some packages, the default dependency solver can take a long time to resolve the environment. Consider running the following commands in order to speed up the process::
+
+        $ conda install -n base conda-libmamba-solver
```

Comment:
This shouldn't be needed for the latest version of conda! Libmamba comes by default, I think it still needs to be set as the default solver though.

Reply:
I don't think it's installed by default. Running only the solver line gives me:
```
CondaValueError: You have chosen a non-default solver backend (libmamba) but it was not recognized. Choose one of: classic
```